src/controllers/v1/attachment/__tests__/index.test.ts

```diff
@@ -0,0 +1,304 @@
+import { expect } from 'chai'
+import sinon from 'sinon'
+import mockdate from 'mockdate'
```

Comment:
If we're just using this to set `Date.now()` is there a reason we can't use https://sinonjs.org/releases/v17/fake-timers/?

Reply:
to be fair was not aware of sinon timers, module has been removed from package JSON as well along with other updates to reflect on fixtures